### ISSUE TITLE
The EventHub.Web app is throwing an error when I try to run it

### ISSUE THREAD

**Issue by justinfraz** [abpframework/eventhub#133]
![EventHub Web - Run error message](https://github.com/abpframework/eventhub/assets/26783435/f32d6e7d-5681-4257-ab94-376ea63d92c2)

**Comment by EngincanV** [comment]
Hi, EventHub is a distributed application. So, you need to run the `EventHub.HttpApi.Host` project before running the `EventHub.Web` application.

You can refer to the [README.md](https://github.com/abpframework/eventhub#how-to-run).

**Comment by justinfraz** [comment]
But when I ran the EventHub.HttpApi.Host project, this is the error message that I got:

![EventHub HttpApi Host error message](https://github.com/abpframework/eventhub/assets/26783435/59cab0f4-c443-4e4f-8cc0-ab0b2ca4121e)

**Comment by justinfraz** [comment]
Yes, I have performed these steps before running the db migrator project:

Execute dotnet build /graphBuild command in the root folder of the solution.
Execute etc/docker/up.ps1 to run the depending services.

**Comment by EngincanV** [comment]
I tried just before running the DbMigrator project and it ran as expected:

![image](https://github.com/abpframework/eventhub/assets/43685404/d0dd4b59-d375-4df4-81a6-35a307a48db7)

You can check your configurations, ensure docker containers are up and running, and also ensure Redis or Postgresql are not running on your PC locally.

**Comment by EngincanV** [comment]
I could successfully run both projects as can be seen in the following figure:

![image](https://github.com/abpframework/eventhub/assets/43685404/af8bc8cf-c470-4ce1-8729-59ca67f94fed)

It seems you have some misconfigurations on your local. You can check your configurations, ensure docker containers are up and running, and also ensure Redis or Postgresql are not running on your PC locally.
